# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../elementary/no_themes.patch.cross
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

data/* needs to run edje_cc

--- ./Makefile.am.orig	2010-04-29 07:15:13.000000000 -0400
+++ ./Makefile.am	2010-04-29 07:15:33.000000000 -0400
@@ -1,4 +1,4 @@
-SUBDIRS = src data doc
+SUBDIRS = src doc
 
 ACLOCAL_AMFLAGS = -I m4
 
